src/ReachSets/ContinuousPost/BFFPSV18/BFFPSV18.jl

```diff
@@ -235,23 +235,29 @@ function init!(𝒫::BFFPSV18, 𝑆::AbstractSystem, 𝑂::Options)
 end
 
 """
-    post(𝒫::BFFPSV18, 𝑆::AbstractSystem, invariant, 𝑂::Options)
+    post(𝒫::BFFPSV18, 𝑆::AbstractSystem, 𝑂::Options)
 
 Calculate the reachable states of the given initial value problem using `BFFPSV18`.
 
 ### Input
 
 - `𝒫` -- post operator of type `BFFPSV18`
 - `𝑆` -- sytem, initial value problem for a continuous ODE
-- `invariant` -- constraint invariant on the mode
 - `𝑂` -- algorithm-specific options
 """
-function post(𝒫::BFFPSV18, 𝑆::AbstractSystem, invariant, 𝑂_input::Options)
+function post(𝒫::BFFPSV18, 𝑆::AbstractSystem, 𝑂_input::Options)
     𝑂 = TwoLayerOptions(merge(𝑂_input, 𝒫.options.specified), 𝒫.options.defaults)
 
     # convert matrix
     system = matrix_conversion(𝑆, 𝑂)
 
+    # apply transformation to the invariant
+    if hasmethod(stateset, Tuple{typeof(𝑆.s)})
+        invariant = 𝑆.s.X
+    else
```

Comment:
What about defining a helper method `invariant(::CF)` in for our `normalize`d systems?